AutomatedTesting/Gem/PythonTests/assetpipeline/metadata_relocation_tests/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+if(PAL_TRAIT_BUILD_TESTS_SUPPORTED AND PAL_TRAIT_BUILD_HOST_TOOLS)

```

Comment:
I recommend adding the blocks of logic for periodic and sandbox while you're here. It's easy to add the sandbox/periodic flag to a test when you want to move it, and forget to check that the associated cmake file has that flag set.

Reply:
Added